libraries/lib-time-and-pitch/FormantShifterLoggerInterface.h

```diff
@@ -0,0 +1,40 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  FormantShifterLoggerInterface.h
+
+  A visualization helper for the development of the formant shifter.
+
+  Matthieu Hodgkinson
+
+**********************************************************************/
+#pragma once
+
+#include <complex>
+#include <functional>
+
+class FormantShifterLoggerInterface
```

Comment:
Do you plan to have different implementation?

Reply:
No. It ended up being a way for me to expose only relevant methods to `FormantShifter` and the rest (`NewSamplesComing` and override getters) to whoever integrates logger and shifter together (`StaffPadTimeAndPitch`).

Reply:
Update: after some clean-up there now is an alternative, dummy implementation that's injected for production.